homeassistant/components/cover/zwave.py

```diff
@@ -110,24 +111,43 @@ class ZwaveGarageDoor(zwave.ZWaveDeviceEntity, CoverDevice):
     def __init__(self, values):
         """Initialize the zwave garage door."""
         ZWaveDeviceEntity.__init__(self, values, DOMAIN)
+        self._door_state_list = None
+        self._door_state = None
         self.update_properties()
 
     def update_properties(self):
         """Handle data changes for node values."""
-        self._state = self.values.primary.data
+        self._door_state = self.values.primary.data
+        door_state_list = self.values.primary.data_items
+        if door_state_list:
+            self._door_state_list = list(door_state_list)
```

Comment:
So something like this?

STATE_OPEN = 'open'
STATE_OPENING = 'opening'
STATE_CLOSED = 'closed'
STATE_CLOSING = 'closing'
STATE_STOPPED = 'stopped'